database/migrations/2020_11_04_144136_add_index_to_user_id_on_comments.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIndexToUserIdOnComments extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('comments', function (Blueprint $table) {
+            $table->index('user_id');
```

Comment:
This index should be `user_id, created_at desc, id desc` because of the listing order (see `2020_09_09_042555_update_comments_index`).